src/index.css

```diff
@@ -352,7 +390,48 @@ button {
   font-size: 12px;
   cursor: pointer;
 }
+.assets-table {
+  display: flex;
+  flex-direction: column;
+  row-gap: 0.5em;
+  width: 100%;
+}
 
+.assets-table-header {
+  display: flex;
+  position: relative;
+  flex-direction: row;
+  height: 50px;
+  width: 100%;
+  border: 1px solid #323232;
+  border-radius: 6px;
+  background-color: #323232;
+  align-items: center;
+}
+.assets-table-item {
+  display: flex;
+  flex-direction: row;
+  position: absolute;
+  justify-content: center;
+  align-items: center;
+  color: var(--text-white);
+  width: auto;
+}
+.assets-table-item:nth-child(1) {
+  left: 50px;
+}
+.assets-table-item:nth-child(2) {
+  left: 25%;
+}
+.assets-table-item:nth-child(3) {
+  left: 50%;
+}
+.assets-table-item:nth-child(4) {
+  left: 75%;
+}
+.assets-table-item:nth-child(5) {
+  right: 10px;
+}
```

Comment:
avoid hard coding table item sizes, it's brittle and likely to make it difficult to update should a new column or table design change